bridge/evm/contracts/utils/BridgeUtils.sol

```diff
@@ -127,6 +127,9 @@ library BridgeUtils {
         pure
         returns (uint64)
     {
+        // ensure provided amount is greater than 0
+        require(amount > 0, "BridgeUtils: Insufficient amount provided");
```

Comment:
you could put it inside the `if` if you cared, not terribly important

Reply:
done